### ISSUE TITLE
train

### ISSUE THREAD

**Issue by hundunshu** [wangguanan/HOReID#17]
I trained according to the steps, but it didn't reach the accuracy in the paper. What's the reason? I trained on a 2080ti 
result：
 Time: 2020-12-29 19:19:33,  base, Dataset: Duke  
mAP: 0.40751378988429915 
Rank: [0.49457014 0.57647059 0.61809955 ... 1.         1.         1.        ]
Time: 2020-12-29 19:27:35,  base+gcn, Dataset: Duke  
mAP: 0.40423225383235895 
Rank: [0.49095023 0.57330317 0.6199095  ... 1.         1.         1.        ]
Time: 2020-12-29 19:34:59,  base+gcn+gm, Dataset: Duke  
mAP: 0.4047804871692115 
Rank: [0.50090498 0.5719457  0.61674208 ... 1.         1.         1.        ]

**Comment by chenxiangzZ** [comment]
you can provide more detail about yout training ,generally, the result would be higher than the paper's

**Comment by hundunshu** [comment]
> 
> 
> you can provide more detail about yout training ,generally, the result would be higher than the paper's

Namespace(auto_resume_training_from_lastest_steps=True, base_learning_rate=0.00035, branch_num=14, cuda='cuda', duke_path='/home/lab/wlb/S-Dataset/Occluded_Dataset/Occluded_Duke/Occluded_Duke', gcn_lr_scale=0.1, gcn_scale=20.0, gm_lr_scale=1.0, image_size=[256, 128], k=4, margin=0.3, max_save_model_num=1, milestones=[40, 70], mode='train', norm_scale=10.0, output_path='./results', p=16, pid_num=702, resume_test_epoch=0, resume_test_path='', resume_visualize_epoch=0, resume_visualize_path='', total_train_epochs=120, train_dataset='duke', use_gm_after=20, ver_alpha=0.5, ver_in_scale=10.0, ver_lr_scale=1.0, ver_topk=1, weight_decay=0.0005, weight_global_feature=1.0, weight_p_loss=1.0, weight_ver_loss=0.1)

**Comment by wangguanan** [comment]
could you please try  to straightly test with our-pretrained model

the pre-trained model can be downloaded from badudisk as in readme.md
